mozilla-release/mobile/android/app/src/main/res/menu/cliqz_browser_app_menu.xml

```diff
@@ -76,11 +76,11 @@
 
     <item
         android:id="@+id/pin_to_top_sites"
-        android:title="@string/contextmenu_pin_to_top_sites" />
+        android:title="@string/cliqz_contextmenu_pin_to_top_sites" />
```

Comment:
Shall I keep the change in this file (plus menu_large & menu_xlarge)? Because I noticed that even without this change, it works. Where is this file used?

Reply:
Well, those files are used on tablets, I'll say we should change those too.

Reply:
I changed large & xlarge files too!